modules/apps/object/object-service/src/main/java/com/liferay/object/internal/field/business/type/MultiselectPicklistObjectFieldBusinessType.java

```diff
@@ -100,6 +102,29 @@ public PropertyDefinition.PropertyType getPropertyType() {
 		return PropertyDefinition.PropertyType.TEXT;
 	}
 
+	@Override
+	public Object getValue(
```

Comment:
I miss a test where we are using this method, not sure if a unit test or an integration test (for example, the REST API as I guess the POST should work the same way)

Reply:
added a test, run it with gw testIntegration --tests MultiselectPicklistObjectFieldBusinessTypeTest